File: trail1/2의 거듭제곱수/pow-of-2.cpp
#include <iostream>
using namespace std;

int main() {
    int N;
    cin >> N;
    int cnt = 0;
    while (N > 2) {
        N /= 2;
        cnt++;
    }
    cout << cnt + 1 << " ";
    return 0;
}

File: trail1/2의 거듭제곱수/pow-of-2.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        int N = int.Parse(Console.ReadLine());
        int cnt = 0;
        while (N > 2) {
            N /= 2;
            cnt++;
        }
        Console.WriteLine(cnt+1);
    }
}
